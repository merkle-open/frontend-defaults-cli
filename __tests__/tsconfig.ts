/// <reference types="@types/jest" />
import { apiIt } from './shared';

describe('tsconfig', () => {
	it('default', async () =>
		await apiIt('tsconfig', {
			ts: true,
		}));
});
