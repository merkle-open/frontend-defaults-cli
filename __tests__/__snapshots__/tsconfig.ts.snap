// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tsconfig default 1`] = `
<div
  dangerouslySetInnerHTML={
    Object {
      "__html": "{
  \\"devDependencies\\": {
    \\"@namics/ts-config\\": \\"0.4.1\\",
    \\"typescript\\": \\"3.6.4\\"
  }
}",
    }
  }
/>
`;

exports[`tsconfig default 2`] = `
<div
  dangerouslySetInnerHTML={
    Object {
      "__html": "{
	\\"extends\\": \\"@namics/ts-config\\",
	\\"exclude\\": [\\"dist\\", \\"build\\", \\"node_modules\\", \\"__tests__\\"]
}
",
    }
  }
/>
`;

exports[`tsconfig default 3`] = `
<div
  dangerouslySetInnerHTML={
    Object {
      "__html": "package.json,tsconfig.json",
    }
  }
/>
`;
